### PR TITLE
Add system tests for nidigital utility methods

### PR DESCRIPTION
**Pull request by sbethur**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

~- [ ] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~

- [x] I've added tests applicable for this pull request

### What does this Pull Request accomplish?

Add system tests for nidigital utility methods. Tests added:
- test_reset
- test_reset_device
- test_self_test
- test_get_error
- test_clear_error
- test_self_calibrate

### List issues fixed by this Pull Request below, if any.

None

### What testing has been done?

Added and executed new system tests

### CODE REVIEW

File: src/nidigital/system_tests/test_system_nidigital.py [hunk 1]
```diff
@@ -20,6 +20,44 @@ def multi_instrument_session():
...
+    assert multi_instrument_session.selected_function == nidigital.SelectedFunction.PPMU
+    multi_instrument_session.reset_device()
+    assert multi_instrument_session.selected_function == nidigital.SelectedFunction.DISCONNECT
+
+
+def test_self_test(multi_instrument_session):
+    multi_instrument_session.self_test()
+
+
+def test_get_error(multi_instrument_session):
```
Comment (marcoskirsch) [inline src/nidigital/system_tests/test_system_nidigital.py:41]: shouldn't this be private?

File: src/nidigital/system_tests/test_system_nidigital.py [hunk 2]
```diff
@@ -20,6 +20,44 @@ def multi_instrument_session():
...
+    try:
+        multi_instrument_session.supported_instrument_models = ''
+        assert False
+    except nidigital.Error as e:
+        assert e.code == -1074135027
+        assert e.description.find('Attribute is read-only.') != -1
+
+
+def test_clear_error(multi_instrument_session):
```
Comment (marcoskirsch) [inline src/nidigital/system_tests/test_system_nidigital.py:50]: shouldn't this be private?

Comment (sbethur) [reply src/nidigital/system_tests/test_system_nidigital.py:50]: It should actually be removed completely - #1366.

File: src/nidigital/system_tests/test_system_nidigital.py (hunk 1 above)
Comment (sbethur) [reply src/nidigital/system_tests/test_system_nidigital.py:41]: `_get_error()` is private. This test is to verify nidigital module bundles the error code and description correctly in the exception object it throws.
Test is similar to what we [have](https://github.com/ni/nimi-python/blob/7911a1400de731e03b620ee9492599aa19f67dcc/src/nidcpower/system_tests/test_system_nidcpower.py#L54) as part of other system tests.

**Pull request merged**